data/json/items/armor.json

```diff
@@ -706,64 +706,58 @@
     "flags": [ "BELTED", "WATER_FRIENDLY" ]
   },
   {
-    "id": "badge_cybercop",
+    "abstract": "badge_abstract",
     "type": "ARMOR",
-    "name": "cybercop shield",
-    "description": "A chromed silver shield identifies the wearer as an authority even the cop-killers couldn't kill.",
+    "name": "badge template",
+    "description": "This is a template for police badges.  If found in a game it is a bug.",
     "weight": 60,
     "volume": 0,
     "price": 5000,
     "material": [ "silver" ],
     "symbol": "[",
-    "color": "yellow"
+    "color": "yellow",
+    "covers": [ "TORSO" ],
+    "coverage": 2,
+    "material_thickness": 2,
+    "flags": [ "ALLOWS_NATURAL_ATTACKS", "OUTER", "OVERSIZE", "WATER_FRIENDLY" ]
+  },
+  {
+    "id": "badge_cybercop",
+    "copy-from": "badge_abstract",
+    "type": "ARMOR",
+    "name": "cybercop badge",
+    "description": "A chromed silver shield identifies the wearer as an authority even the cop-killers couldn't kill."
   },
   {
     "id": "badge_deputy",
+    "copy-from": "badge_abstract",
     "type": "ARMOR",
     "name": "deputy badge",
-    "description": "A tarnished silver star gives an air of authority to the wearer.",
-    "weight": 60,
-    "volume": 0,
-    "price": 5000,
-    "material": [ "silver" ],
-    "symbol": "[",
-    "color": "yellow"
+    "description": "A tarnished silver star gives an air of authority to the wearer."
   },
   {
     "id": "badge_detective",
+    "copy-from": "badge_abstract",
     "type": "ARMOR",
     "name": "detective badge",
     "description": "A tarnished golden shield gives an air of authority to the wearer.",
-    "weight": 60,
-    "volume": 0,
-    "price": 5000,
     "material": [ "gold" ],
-    "symbol": "[",
-    "color": "yellow"
+    "proportional": { "weight": 2 }
   },
   {
     "id": "badge_marshal",
+    "copy-from": "badge_abstract",
     "type": "ARMOR",
     "name": "marshal badge",
     "description": "A bright silver star strikes fear into the eyes of ne'er-do-wells.",
-    "weight": 60,
-    "volume": 0,
-    "price": 10000,
-    "material": [ "silver" ],
-    "symbol": "[",
-    "color": "yellow"
+    "proportional": { "price": 2 }
```

Comment:
Yeah, I didn't address that explicitly as it was originally in the game files (10,000 instead of 5,000). It seems the only badge that you can't start at, guess that's what it makes it special (which explains why it's *marshal* badge).